phpunit.xml.dist

```diff
@@ -7,10 +7,11 @@
          convertWarningsToExceptions="true"
          processIsolation="false"
          stopOnFailure="false"
-         syntaxCheck="false"
          bootstrap="tests/bootstrap.php">
 
     <testsuite name="Config Migration Tool Test Suite">
-        <directory>tests/Keboola</directory>
+        <directory>./tests/Keboola</directory>
+        <exclude>./tests/Keboola/ConfigMigrationTool/Migrations/KeboolaGoodDataWriterMigrationTest.php</exclude>
```

Comment:
proc je to vyhozeny?

Reply:
Pretoze to su testy rozdelanych migraci pre GoodData writer, ktore failovali.

Reply:
a nemuzem je smazat z masteru, kdyz jsou rozdelany?

Reply:
Tie testy? To by sme asi mohli.

Reply:
Ony ale mimochodem fungovaly, já je nerozbil. ;o) Ale upravuju a dokončuju je tady: https://github.com/keboola/config-migration-tool/pull/29 tak asi můžem